AmplifyPlugins/Auth/Sources/AWSCognitoAuthPlugin/AWSCognitoAuthPlugin+Configure.swift

```diff
@@ -113,7 +113,9 @@ extension AWSCognitoAuthPlugin {
     }
 
     private func makeURLSession() -> URLSession {
-        return URLSession.shared
+        let configuration = URLSessionConfiguration.default
```

Comment:
Oh, we're modifying the configuration we technically don't own, right? Any chance we can do this instead so that we're not messing with the configuration the host app (or other libraries) may be using?
```
        let configuration = URLSessionConfiguration.default.mutableCopy()
```

Reply:
Discussed offline, thanks!